chart/templates/config.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
   annotations: {{- include "harnesscommon.tplvalues.render" ( dict "value" .Values.global.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 data:
+ {{- include "harnesscommon.tplvalues.render" ( dict "value" .Values.config "context" $) | nindent 2}}
   pool: |-
```

Comment:
better to put the mount file in another configmap, so files are not mounted as pool var is not mounted as envVar

Reply:
resolved.